k6/createPosts.js

```diff
@@ -0,0 +1,63 @@
+import http from 'k6/http';
+
+const config = JSON.parse(open('../config/config.json'));
+const response = JSON.parse(open('../temp_store.json'));
+
+export const options = {
+    vus: config.PostsConfiguration.Count,
+    duration: config.PostsConfiguration.Duration,
+}
+
+function getRandomMessage(wordsCount, wordLength) {
+	let message = '';
+    const characterSet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+	let words = 0;
+    wordsCount = Math.floor(Math.random() * wordsCount) + 1;
+    while (words < wordsCount) {
+        let count = 0;
+        wordLength = Math.floor(Math.random() * wordLength) + 1;
+        while (count < wordLength) {
+            message += characterSet.charAt(Math.floor(Math.random() * characterSet.length));
+            count += 1;
```

Comment:
```suggestion
            count++;
```
Use this instead of `+= 1` everywhere.